src/backend/distributed/utils/metadata_cache.c

```diff
@@ -3565,7 +3757,8 @@ TupleToShardInterval(HeapTuple heapTuple, TupleDesc tupleDescriptor, Oid interva
 	 */
 	heap_deform_tuple(heapTuple, tupleDescriptor, datumArray, isNullArray);
 
-	relationId = DatumGetObjectId(datumArray[Anum_pg_dist_shard_logicalrelid - 1]);
+	relationId = DatumGetObjectId(datumArray[Anum_pg_dist_shard_logicalrelid -
+											 1]);
```

Comment:
Why are all of these reindent changes? Do you have different config for reindent by any chance?

Reply:
I think my version is different